packages/php-wasm/universal/src/lib/php-request-handler.ts

```diff
@@ -23,6 +23,25 @@ export type RewriteRule = {
 	replacement: string;
 };
 
+export type FileNotFoundToResponse = {
+	type: 'response';
+	response: PHPResponse;
+};
+export type FileNotFoundToInternalRedirect = {
+	type: 'internal-redirect';
+	uri: string;
+};
+export type FileNotFoundTo404 = { type: '404' };
+
+export type FileNotFoundAction =
```

Comment:
Lovely naming

Reply:
> Lovely naming

❤️  I think so too. The first names were not so good, and it was satisfying to find something better.